public/pages/VisualCreatePolicy/components/Transition/Transition.tsx

```diff
@@ -31,7 +32,8 @@ interface TransitionProps {
 
 const Transition = ({ uiTransition, onChangeTransition }: TransitionProps) => {
   // We currently only support one transition condition
-  const conditionType = Object.keys(uiTransition.transition?.conditions || []).pop() || "min_index_age";
+  const conditionType = Object.keys(uiTransition.transition?.conditions || []).pop() || "none";
+  const conditions = uiTransition.transition.conditions;
```

Comment:
Confused here, above is using `uiTransition.transition?.`, but no `?` down here

Reply:
Hmm the `?` might be leftover from earlier edits.. it won't do anything, can remove it in follow up PR so tests aren't rerun again for 30 minutes..

But, the interfaces are:

```
export interface UITransition {
  transition: Transition;
  id: string;
}

export interface Transition {
  state_name: string;
  conditions?: Condition;
}
```

So transition should not be possibly undefined, but the conditions might be undefined.